wrench/src/yaml_frame_writer.rs

```diff
@@ -492,16 +492,15 @@ impl YamlFrameWriter {
     }
 
     fn path_for_image(&mut self, key: ImageKey) -> Option<PathBuf> {
-        if let Some(ref mut data) = self.images.get_mut(&key) {
-            if data.path.is_some() {
-                return data.path.clone();
-            }
-        } else {
+        let data = self.images.get_mut(&key);
+        if data.is_none() {
```

Comment:
How about something like:

```
match self.images.get_mut(&key) {
    Some(data) => {
       ...
    }
    None => {
      return;
    }
```

Reply:
I wanted to avoid indenting all the code after the changed lines, and still leave the error handling at the top.

I don't object to changing it if that's what you prefer.

Reply:
Yep, let's change it to that style and then this should be good to merge. Thanks!

Reply:
Actually, having a `match` there doesn't force indentation:
```rust
let data = match self.images.get_mut(&key) {
  Some(data) => data,
  None => return;
};
```

Reply:
@kvark Ooh, nice. Will do that.